docs/semgrep-app/integrations.md

```diff
@@ -1,19 +1,163 @@
 
-The Jira integration allows you to create tickets directly from the Findings page with relevant info about a particular finding.
+Currently, Semgrep App integrates with the following tools:
 
-![Setting up the Jira integration](../img/jira-integrations-page.png)
-![Creating a Jira ticket from the Findings page](../img/jira-findings-page.png)
+| Tool | Tier availability |
+| ---- | ---------------- |
+| Slack | Community (Free) |
+| Email | Community (Free) |
+| Jira | Team/Enterprise |
+| Amazon S3 | Team/Enterprise |
+| Webhook | Team/Enterprise |
+
+## Finding available integrations
+
+To find available integrations for Semgrep App, follow these steps:
+
+1. Sign in to your [Semgrep App account](https://semgrep.dev/).
+2. Click **Settings**.
+3. Click **Integrations**.
+
+![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)<br />
+
+## Managing integrations
+
+To view, disable, or enable your saved integration channels:
+
+1. In the **Integrations** tab, click the **gear** icon within the **Rule board**.
+2. Select the toggles to turn notifications on or off for each channel.
+
+![Screenshot of the Semgrep rule board with integrations](../img/integration-ruleboard.png)<br />
+
+## Integrating various third-party tools
+
+This section describes how to integrate Semgrep App into particular third-party tools.
+
+### Slack
+
+Integrating with Slack allows Semgrep to send findings to a channel on your Slack workspace.
+
+To set it up, create an app within Slack and enable that app's **Incoming Webhooks** feature. By creating a webhook, you'll have a URL that serves as Semgrep's endpoint to your Slack workplace. Follow the guide below to create the app and webook, and integrate with Semgrep:
+
+1. Log in to your Slack account. Enter your login details in [Slack workspace](https://slack.com/workspace-signin).
+2. Create an app in your Slack workplace by following this link: [Slack apps](https://api.slack.com/apps?new_app=1).
+3. Select **From scratch** and fill in the fields for your app's name and workspace.
+![Screenshot of a Slack form to fill out new app details](../img/slack-create-app-name.png)
+4. You will be redirected to your new app's settings page. Alternatively, you can find this page by viewing your apps [here](https://api.slack.com/apps) and selecting your integration app from the menu.
+5. Click on **Incoming Webhooks** on the left menu, and then enable the toggle.
+6. After you have enabled **Incoming Webhooks**, follow one of these steps:
+    - Generate a **Webhook URL**.
+    - Request to add a new webhook from a workspace owner.
+7. After you've obtained the Webhook URL, return to Semgrep and create a new Slack integration. Name the integration and copy-paste the URL into the **Slack URL** field. You may opt to select the **Inventory** flag to include notifications about Code Asset Inventory findings.
```

Comment:
Do we have any docs about Code Asset Inventory? If we have them, would be nice to link here!

Reply:
There's no mention of CAI in the docs at all. I'll add a task for it :)